File: lecture10/exercise10-A/template.py
#!/usr/bin/env python3

# Exercise 10-A: Palindromic Permutations

import collections
import sys

# Functions

def is_palindromic(s):
    # TODO
    return False

# Main Execution

def main():
    for word in sys.stdin:
        print('Yes' if is_palindromic(word.rstrip()) else 'No')

if __name__ == '__main__':
    main()

File: lecture10/exercise10-A/template.cpp
// Exercise 10-A: Palindromic Permutations

#include <iostream>
#include <string>
#include <unordered_set>

using namespace std;

// Functions

bool is_palindromic(string &s) {
    return false;
}

// Main execution

int main(int argc, char *argv[]) {
    string word;

    while (cin >> word) {
    	cout << (is_palindromic(word) ? "Yes" : "No") << endl;
    }

    return 0;
}
